src/main/resources/deployments/defaultDeployment.properties

```diff
@@ -198,6 +198,15 @@ dmptool.base.url=https://dmptool-stg.cdlib.org
 # For testing locally, e.g. https://92832de0.ngrok.io, defaults to properties.getServerUrl()
 dmptool.callback.base.url=
 
+# Client ID / Secret from Bitwarden
+dcd.client.id=kW1WtsEeW7b9LugvT-jq9148gmkTPDeFFZe4NM9mV1k
+dcd.client.secret=Q5QHqNn91Q2tvJwsqkRDorAS9i6Mg685GLY6TYLBWH22MITQ4CvOnRx3kQ43IKAr
+dcd.base.url=https://data.mendeley.com
+# For testing locally, e.g. https://researchspace.eu.ngrok.io, defaults to properties.getServerUrl()
+dcd.callback.base.url=https://researchspace.eu.ngrok.io
```

Comment:
make sure this will be blank before merge, so it will get the base server address by default

Reply:
done